app/models/vm/operations.rb

```diff
@@ -9,38 +9,17 @@ module Vm::Operations
   include_concern 'Lifecycle'
 
   included do
-    supports :vnc_console do
-      message = "VNC Console not supported"
-      if vendor == 'vmware'
-        unsupported_reason_add(:vnc_console, message) unless ext_management_system.present? && console_supports_type?('VNC')
-      elsif !console_supported?('VNC')
-        unsupported_reason_add(:vnc_console, message)
-      end
-    end
-
-    supports :mks_console do
-      message = "WebMKS Console not supported"
-      if vendor != 'vmware'
-        unsupported_reason_add(:mks_console, message)
-      elsif console_supported?('WEBMKS') && !console_supports_type?('WebMKS')
-        unsupported_reason_add(:mks_console, message)
-      end
+    supports :html5_console do
+      consup = %w[vnc webmks spice].map { |type| send(:console_supported?, type) }.reduce(:|)
```

Comment:
Could this just be `%w[vnc webmks spice].any? { |type| send(:console_supported?, type) }`